PipelinesTasks/PRMetrics/src/task/tests/pullRequests/pullRequestComments.spec.ts

```diff
@@ -300,23 +300,34 @@ describe('pullRequestComments.ts', (): void => {
       verify(logger.logDebug('* PullRequestComments.getNoReviewRequiredCommentData()')).twice()
     })
 
-    it('should continue when no comment content is present', async (): Promise<void> => {
-      // Arrange
-      when(azureReposInvoker.getCommentThreads()).thenResolve([{ comments: [{}] }])
-      when(codeMetrics.getFilesNotRequiringReview()).thenResolve(['file.ts'])
-      const pullRequestComments: PullRequestComments = new PullRequestComments(instance(azureReposInvoker), instance(codeMetrics), instance(inputs), instance(logger), instance(taskLibWrapper))
+    async.each(
+      [
+        [[{ comments: [{}] }], null, 1, 0],
+        [[{ comments: [validGitPullRequestCommentThread.comments![0]!, {}], id: 1 }], 1, 1, 0],
+        [[{ threadContext: { filePath: '/file.ts' }, comments: [{}] }], null, 0, 1],
+        [[validGitPullRequestCommentThread, { comments: [{}] }], null, 1, 1],
+        [[validGitPullRequestCommentThread, { comments: [validGitPullRequestCommentThread.comments![0]!, {}], id: 1 }], 1, 1, 1],
+        [[validGitPullRequestCommentThread, { threadContext: { filePath: '/file.ts' }, comments: [{}] }], null, 0, 2]
+      ], (data: [GitPullRequestCommentThread[], number | null, number, number]): void => {
+        it(`should continue when no comment content is present in payload '${JSON.stringify(data[0])}'`, async (): Promise<void> => {
+          // Arrange
+          when(azureReposInvoker.getCommentThreads()).thenResolve(data[0])
+          when(codeMetrics.getFilesNotRequiringReview()).thenResolve(['file.ts'])
+          const pullRequestComments: PullRequestComments = new PullRequestComments(instance(azureReposInvoker), instance(codeMetrics), instance(inputs), instance(logger), instance(taskLibWrapper))
 
-      // Act
-      const result: PullRequestCommentsData = await pullRequestComments.getCommentData(1)
+          // Act
+          const result: PullRequestCommentsData = await pullRequestComments.getCommentData(1)
```

Comment:
Was this whitespace change intended?

Reply:
Yes, I needed to increase the indentation after wrapping the method within the `async.each` block. Thanks for verifying.